### PR TITLE
FIX: Pause QRCode during Export

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/zwnkezcem8tr07kyrdh0yyjq58?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/8254)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/pg3j1kavwk67aztp0qqbam6a08?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/8255)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

File: screen/send/psbtMultisigQRCode.js [hunk 1]
```diff
@@ -64,15 +75,23 @@ const PsbtMultisigQRCode = () => {
...
 
   const exportPSBT = () => {
+    dynamicQRCode.current?.stopAutoMove();
     setIsLoading(true);
-    setTimeout(() => fs.writeFileAndExport(fileName, psbt.toBase64()).finally(() => setIsLoading(false)), 10);
+    setTimeout(
+      () =>
+        fs.writeFileAndExport(fileName, psbt.toBase64()).finally(() => {
+          setIsLoading(false);
+          dynamicQRCode.current?.startAutoMove();
```
Comment (Overtorment) [inline screen/send/psbtMultisigQRCode.js:84]: feels like you dont need to start it at all if user exported to a file, not sure

**Review by Overtorment — APPROVED** [review]

**Pull request merged**